### ISSUE TITLE
Error Build

### ISSUE THREAD

**Issue by Tofdu31** [Johni0702/mumble-web#56]
Hi Johni0702,

I have anerror when I build your project : 
`ERROR: Service 'mumble-client' failed to build: The command '/bin/sh -c apk del gcc git make g++' returned a non-zero code: 2`

Do you have a solution ?

And i Have : 
```
Step 8/15 : RUN cd /home/node &&     npm install &&     npm run build
 ---> Running in 6db6f2a87fa2
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read https://babeljs.io/env to update!
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated fsevents@1.2.9: One of your dependencies needs to upgrade to fsevents v2: 1) Proper nodejs v10+ support 2) No more fetching binaries from AWS, smaller package size
```

**Comment by Johni0702** [comment]
I'm not familiar with docker. @andreaspeters contributed the Dockerfile.

That command just removes some packages which aren't required at runtime, you should be fine just removing that line from the Dockerfile (presumably at the expense of disk space).

**Comment by Tofdu31** [comment]
OK thank a lot, I write to @andreaspeters 
I will keep you informed as soon as the solution is corrected.

Tof

**Comment by andreaspeters** [comment]
My mistake! Sorry! Change the line to this please.

```
RUN apk del git
```

**Comment by Tofdu31** [comment]
Ok thank,

This is the log of
`Run apk del git`

```
Step 10/15 : RUN apk del git
 ---> Running in f8e3b5819830
WARNING: Ignoring APKINDEX.066df28d.tar.gz: No such file or directory
WARNING: Ignoring APKINDEX.b53994b4.tar.gz: No such file or directory
WARNING: Ignoring APKINDEX.cbed5262.tar.gz: No such file or directory
(1/3) Purging git (2.24.0-r0)
(2/3) Purging libcurl (7.67.0-r0)
(3/3) Purging pcre2 (10.33-r0)
Executing busybox-1.31.1-r0.trigger
```
Wath's this warning ?

**Comment by andreaspeters** [comment]
He can not find the archive with the available software packages for alpine.

**Comment by Tofdu31** [comment]
OK thank 👍 

And for this : 
```
Step 8/15 : RUN cd /home/node &&     npm install &&     npm run build
 ---> Running in b4d69d2530ae
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read https://babeljs.io/env to update!
npm WARN deprecated core-js@2.6.10: core-js@<3.0 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated fsevents@1.2.9: One of your dependencies needs to upgrade to fsevents v2: 1) Proper nodejs v10+ support 2) No more fetching binaries from AWS, smaller package size
```

**Comment by andreaspeters** [comment]
This is something @Johni0702 have to care in his package.json. :-)

**Comment by Tofdu31** [comment]
OK thank !

**Comment by JuniorJPDJ** [comment]
Could you guys please fix this dockerfile in repository?

**Comment by andreaspeters** [comment]
Hi @JuniorJPDJ ! I fix it with PR #64. I'm sure @Johni0702 will merge it soon.
